tsconfig.json

```diff
@@ -13,9 +13,13 @@
     "sourceMap": true,
     "rootDir": "."
   },
+  "include": [
```

Comment:
I did and it works. I also looked at the `tsconfig` [docs ](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) just in case and the `exclude` section can be removed without any issue.